docs/resources/appdata_dsource.md

```diff
@@ -0,0 +1,176 @@
+# Resource: <resource name> delphix_appdata_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Virtualization Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+
+The dSource resource allows terraform to CREATE, READ and DELETE dSources. 
+Update operation is not supported. The supported parameters are listed below.
```

Comment:
"The dSource resource allows Terraform to `apply` and `destroy` Delphix dSources."

CREATE, READ, and DELETE aren't real Terraform operations. If there is another Terraform operation that I missed though, please add it.

Reply:
Similarly "Update operation" isn't a real operation either. Our documentation should apply directly towards Terraform.

Reply:
We have used the same CREATE, READ, and DELETE in vdb.md also , shall we update that as well ?

Reply:
@nick-mathison The apply and destroy aren't really what a customer would be interested in. 
I think the more imporant information for us to show is what Delphix operations are supported via terraform. For instance, for a dsource does this provider support Linking (creation) and Delete or Updates are also supported. 
This goes across the board. LMK your thoughts.

Reply:
This documentation lives on the Terraform website, thus, , I think we should be speaking from a Terraform point of view. Adding create and delete terminology is 

"The dSource resource allows Terraform to create and delete Delphix dSources. This specifically enables the `apply` and `destroy` Terraform commands. Modification of existing dSource resources via the `apply` command is not supported. All supported parameters are listed below."

Reply:
Suggestion for VDBs

"The VDB resource allows Terraform to create, update, and delete Delphix VDBs. This specifically enables the `apply` and `destroy` Terraform commands. All supported parameters are listed below."